raft.c

```diff
@@ -665,6 +726,15 @@ static char *raftMembershipInfoString(raft_server_t *raft)
     return buf;
 }
 
+static void handleTransferLeaderComplete(raft_server_t *raft, raft_transfer_state_e state);
+
+static void raftNotifyTransferEvent(raft_server_t *raft, void *user_data, raft_transfer_state_e state)
+{
+    if (redis_raft.transfer_req) {
+        handleTransferLeaderComplete(raft, state);
```

Comment:
Same "if check" is also inside handleTransferLeaderComplete() with some error logic but it will not reach there. Btw, maybe just copy handleTransferLeaderComplete() contents into this function?

Reply:
removed

Reply:
ok, i'm unsure.  I like the fact that the raftreq "handlers" and the callback are separated, but I can combine that if desired.  but you are right, the check isn't neccessary (and in fact counterproductive)

Reply:
Okay, however you want.